apps/web/src/pages/DNDDigitalCareersPage/DNDDigitalCareersPage.tsx

```diff
@@ -0,0 +1,533 @@
+                  id: "Y9Tig+",
+                  description: "Description of the data analyst role",
+                })}
+              </p>
+            </Card.GridItem>
+          </Card.Grid>
+          <Card.Separator className="my-6" />
+          <div className="flex flex-wrap items-center gap-6">
+            <Link
+              mode="solid"
+              href="DSGDigitalUpskilling-GSNPerfectionnementnumerique@forces.gc.ca"
+              external
+            >
+              {intl.formatMessage({
+                defaultMessage: "Contact the Innovation Corps team",
+                id: "BM8p9E",
+                description:
+                  "Link text to start an email to the innovation corps",
+              })}
+            </Link>
+            <Link mode="inline" href={paths.browsePools()}>
+              {intl.formatMessage({
+                defaultMessage: "Browse jobs on GC Digital Talent",
+                id: "2jApk/",
+                description: "Link text to browse jobs on GC Digital Talent",
+              })}
+            </Link>
+          </div>
+        </Card>
+        <Heading
+          color="secondary"
+          level="h2"
+          icon={ChatBubbleLeftRightIcon}
+          className="font-normal"
+        >
+          {intl.formatMessage({
+            defaultMessage:
+              "Meet DSG at job fairs for students and recent graduates",
+            id: "YBq69c",
+            description: "Heading for DSG job fairs section",
+          })}
+        </Heading>
+        <p className="my-6">
+          {intl.formatMessage({
+            defaultMessage:
+              "Members of the Digital Services Group will be attending job fairs at universities and colleges in the coming months. Stop by to introduce yourself, ask questions, and discuss how you can start your career at DND.",
+            id: "xlwz2n",
+            description: "Lead-in text for list of upcoming job fairs",
+          })}
+        </p>
+        {jobFairs.length > 0 ? (
+          <div className="flex flex-col gap-3">
+            {jobFairs.map((fair) => (
+              <Card key={fair.title}>
+                <Heading level="h3" size="h5" className="mt-0 mb-3">
+                  <Link
+                    external
+                    newTab
+                    inHeading
+                    mode="inline"
+                    href={fair.href[locale]}
+                  >
+                    {fair.title}
+                  </Link>
+                </Heading>
+                <div className="flex flex-col flex-wrap gap-3 text-sm sm:flex-row">
+                  <span>
+                    {intl.formatMessage(commonMessages.date) +
+                      intl.formatMessage(commonMessages.dividingColon)}
+                    <span className="font-bold">{fair.date}</span>
+                  </span>
```

Comment:
Font colour for the date value appears to be a dark gray and not black

<img width="909" height="259" alt="image" src="https://github.com/user-attachments/assets/32075bd5-1b0f-4195-8d64-567676f61364" />